packages/next/client/page-loader.js

```diff
@@ -118,7 +118,7 @@ export default class PageLoader {
     script.crossOrigin = process.crossOrigin
     script.src = url
     script.onerror = () => {
-      const error = new Error(`Error when loading route: ${route}`)
+      const error = new Error(`Error when loading route: ${route}, url: ${url}`)
```

Comment:
This increases the production bundle size. Let's change it to:

```suggestion
      const error = new Error(`Error loading script ${url}`)
```